### ISSUE TITLE
ql:uninstall should accept a list of systems to uninstall

### ISSUE THREAD

**Issue by Symbolics** [quicklisp/quicklisp-client#147]
As it is, systems have to be uninstalled one-by-one
Also, a :remove-dependencies option would be nice for those systems that pull in the entire universe

**Comment by Hexstream** [comment]
Agreed, meanwhile `(mapc #'ql:uninstall list-of-systems)` is a simple workaround.

**Comment by obicons** [comment]
https://github.com/quicklisp/quicklisp-client/pull/164

**Comment by takuspine** [comment]
In the latest version of quicklisp
`(mapc #'ql:uninstall (mapcar #'ql-dist:short-description (ql-dist:installed-systems t)))`
